index.js

```diff
@@ -46,7 +47,9 @@ function loadPrismPlugin(name) {
 function highlight(text, lang) {
 	const prismLang = loadPrismLang(lang);
 	if (prismLang) {
-		return Prism.highlight(text, prismLang);
+		return `<pre class="language-${lang}"><code class="language-${lang}">\n\t${Prism.highlight(text, prismLang)}</code></pre>`;
```

Comment:
Please use `markdownit.options.langPrefix` instead of `language-` to stay compatible to markdown-it (also in line 52).